Assignment10.py

```diff
@@ -0,0 +1,7 @@
+list1=[1,2,8,4,2,6,4,10,10,10]
+list2=[]
+for elem in list1:
+    if(list2.__contains__(elem)==False):
```

Comment:
== False is not needed.

if (elem in list2) 
OR
if (list2.__contains__(elem)):